assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -622,8 +651,9 @@ end subroutine update_inflation
 !> Uses one of 2 algorithms in references on DART web site to update the 
 !> distribution of inflation:  Anderson 2007, 2009 or Gharamti 2017
 
-subroutine bayes_cov_inflate(ens_size, inf_type, x_p, sigma_p_2, y_o, sigma_o_2, lambda_mean, lambda_sd, &
-   gamma_corr, sd_lower_bound_in, sd_max_change_in, new_cov_inflate, new_cov_inflate_sd)
+subroutine bayes_cov_inflate(ens_size, inf_type, x_p, sigma_p_2, y_o, sigma_o_2, &
```

Comment:
if this is just a line-wrap change, next time could this be deferred until a separate pull request?  i just spent time looking for the change in variable names for this routine.  (i already have 'ignore white space changes' set on.)